psl/psl-core/src/builtin_connectors/mongodb/mongodb_types.rs

```diff
@@ -39,7 +39,7 @@ static DEFAULT_MAPPING: Lazy<HashMap<ScalarType, MongoDbType>> = Lazy::new(|| {
         (ScalarType::Float, MongoDbType::Double),
         (ScalarType::Boolean, MongoDbType::Bool),
         (ScalarType::String, MongoDbType::String),
-        (ScalarType::DateTime, MongoDbType::Timestamp),
+        (ScalarType::DateTime, MongoDbType::Date),
```

Comment:
The change mentioned above 👆 has rippled in unintended ways. Notably, it surfaced an incorrect default native type for the `DateTime` prisma type. This is why I've changed this.